pytest_reportportal/service.py

```diff
@@ -119,6 +121,14 @@ def init_service(self, endpoint, project, uuid, log_batch_size,
             log.debug('The pytest is already initialized')
         return self.RP
 
+    def get_launch_id(self):
+        if (
```

Comment:
Would it be possible to add a docstring and unit tests for that interface?

Reply:
Adding unittests here requires adding mocks for all basic requests for initialiting the service

Should it be done by me?

Reply:
Unit tests should be written by a developer of a code.

Reply:
Just to clarify, any changes in public interfaces are supposed to include unit tests. Usually, they are written by the same person.